### PR TITLE
Code 3312 - add test to ensure pcgen.jar runs

### PR DESCRIPTION
**Pull request by javydreamercsw**
Address issue CODE-3312.

### CODE REVIEW

**Comment by LegacyKing** [comment]
LGTM

File: code/src/itest/pcgen/io/JarTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        assertNotEquals(0, libs.listFiles().length);
+    }
+
+    @Test
+    public void testJar() throws IOException {
+        // Command to create an external process
+        String command = "java -jar pcgen.jar";
+
+        // Running the above command 
+        Runtime run = Runtime.getRuntime();
```
[the remark this answers is not in the document]
Comment (javydreamercsw) [reply code/src/itest/pcgen/io/JarTest.java]: Sure, why not.

File: code/src/itest/pcgen/io/JarTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+ */
+public class JarTest {
+
+    private final static File jar = new File("pcgen.jar");
+    private final static File libs = new File("libs");
+
+    @BeforeClass
+    public static void setup() {
+        // Make sure the jar is in root.
+        assertTrue(jar.exists());
```
[the remark this answers is not in the document]
Comment (javydreamercsw) [reply code/src/itest/pcgen/io/JarTest.java]: It's using assertThat completely necessary? Maybe I'm missing something but not sure what difference does it make.

[the remark this answers is not in the document]
Comment (grimreaper) [reply code/src/itest/pcgen/io/JarTest.java]: Not required: just modern and typesafe. Also better error messages

**Comment by LegacyKing** [comment]
@grimreaper review again please

**Pull request merged**